atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -370,8 +370,14 @@ def get_image_components(self):
                            PostBuildRPMqaPlugin.key)
             return []
 
+        try:
+            sep = PostBuildRPMqaPlugin.sep
+        except AttributeError:
+            # sep instance variable added in Aug 2016
```

Comment:
This implies plugins are from different atomic-reactor releases. Is that a valid scenario?